src/meta/src/telemetry.rs

```diff
@@ -156,6 +156,10 @@ impl MetaTelemetryInfoFetcher {
 #[async_trait::async_trait]
 impl TelemetryInfoFetcher for MetaTelemetryInfoFetcher {
     async fn fetch_telemetry_info(&self) -> TelemetryResult<Option<String>> {
+        // the err here means building cluster on test env, so we don't need to report telemetry
```

Comment:
other nodes get TrackingID from this func, via meta rpc interface. If we do not offer a TrackingID here, the report on other nodes will be disabled.